app/src/main/java/com/github/wnder/Picture.java

```diff
@@ -0,0 +1,33 @@
+package com.github.wnder;
+
+import android.net.Uri;
+
+import com.google.android.gms.maps.model.LatLng;
+
+import java.util.Map;
+
+public interface Picture {
+    /**
+     * Returns the unique ID of the picture
+     * @return The unique ID of the picture
+     */
+    public String getUniqueId();
+
+    /**
+     * Returns the longitude and latitude of the location of the picture
+     * @return the longitude and latitude of the location of the picture
+     */
+    public LatLng getLatLng();
```

Comment:
Location should be used instead of LatLng

Reply:
Done